docs/advisors/checks/mysql_version_eol_57.md

```diff
@@ -0,0 +1,17 @@
+# Percona Server and MySQL 5.7 End Of Life
+## Description
+Percona Server and MySQL server 5.7 are declared End Of Life versions. The deadline is on  Oct 21, 2023. 
```

Comment:
Did you mean Percona Server for MySQL Server 5.7?

Reply:
Both of them